sceptre/stack.py

```diff
@@ -263,17 +265,6 @@ def connection_manager(self):
 
         return self._connection_manager
 
-    @property
-    def sceptre_user_data(self):
-        """Returns sceptre_user_data after ensuring that it is fully resolved.
-
-        :rtype: dict or list or None
-        """
-        if not self._sceptre_user_data_is_resolved:
-            self._sceptre_user_data_is_resolved = True
-            self._resolve_sceptre_user_data()
-        return self._sceptre_user_data
-
```

Comment:
We don't need this any more, since Resolvers handle this situation internally now.